src/split-object/README.md

```diff
@@ -26,3 +26,38 @@ Here's a starter docstring for your solutions. Feel free to rewrite it if that h
  * // -> []
  */
 ```
+
+<!-- BEGIN DOCS -->
+
+<a name="splitObject"></a>
+
+## splitObject ⇒ <code>Array</code>
+
+Splits an object into multiple objects with one key/value pair each.
+The new objects are returned in an array.
+
+This function has no side-effects, the argument object is not modified
+
+**Returns**: <code>Array</code> - Returns a new array with one entry for each key/value pair.
+
+| Param        | Type                | Default         | Description                               |
+| ------------ | ------------------- | --------------- | ----------------------------------------- |
+| [toSeparate] | <code>Object</code> | <code>{}</code> | The object to split into key/value pairs. |
+
+**Example**
+
+```js
+splitObject({ a: 1, b: 2, c: 3 });
+// -> [{ a: 1 }, { b: 2 }, { c: 3 }]
+```
+
+**Example**
+
+```js
+splitObject({});
+// -> []
+```
+
+> Docs generated: Thu May 20 2021, 11:07:35 AM
+
```

Comment:
Hello @mayuri2510 we just missed a few smalls details to develop in the README file such as:
* Strategy
* Implementation
* Use Cases
* Inspiration (if you got inspired in any prev exercise or website that you would like to share)